urdf2webots/parserURDF.py

```diff
@@ -866,6 +863,47 @@ def isRootLink(link, childList):
             return False
     return True
 
+def cleanDummyLinks(linkList, jointList):
```

Comment:
That is maybe more precise:
```suggestion
def removeDummyLinks(linkList, jointList):
```
It's not a clean-up, but a removal, isn't it?